profiler/include/profiler/profile_gemm_blockscale_wp_impl.hpp

```diff
@@ -104,6 +105,10 @@ bool profile_gemm_blockscale_weighpreshuffle_impl(int do_verification,
                                       ? ((K + ScaleBlockK - 1) / ScaleBlockK)
                                       : ((N + ScaleBlockN - 1) / ScaleBlockN);
 
+    ck::utils::validate_gemm_stride<ALayout>(M, K, StrideA, "StrideA");
+    ck::utils::validate_gemm_stride<BLayout>(K, N, StrideB, "StrideB");
+    ck::utils::validate_gemm_stride<BLayout>(M, N, StrideE, "StrideE");
```

Comment:
```suggestion
    ck::utils::validate_gemm_stride<ELayout>(M, N, StrideE, "StrideE");
```

Fix proposed in #2767